.github/workflows/e2e-tests.yml

```diff
@@ -0,0 +1,55 @@
+name: End-to-end tests
+
+# TODO: Once tests are added, run on PRs and commits to development.
+on: workflow_dispatch
```

Comment:
Since there are no e2e tests written yet, there's no point in running this on PRs/commits.

For now, this lets us run the workflow manually.